l10n_ch_payment_slip/payment_slip.py

```diff
@@ -556,6 +564,8 @@ def _draw_bank(self, canvas, print_settings, initial_position, font, bank):
             if not line:
                 continue
             text.textLine(line)
+        line = (bank.zip + '' or '') + ' ' + (bank.city or '')
```

Comment:
My bad I didn't re-read my code... No I meant to write this:
`line = (bank.zip + ' ' or '') + (bank.city or '')`
But your solution is way better. Thank you for the very interesting link :)